crates/bevy_pbr/src/render/pbr_fragment.wgsl

```diff
@@ -420,6 +420,7 @@ fn pbr_input_from_standard_material(
         //
         // This code comes from the `KHR_materials_anisotropy` spec:
         // <https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_anisotropy/README.md#individual-lights>
+#ifdef PBR_ANISOTROPY_TEXTURE_SUPPORTED
```

Comment:
I think this should also be around the texture binding not just it's use.

Reply:
done